website/notifications/emails.py

```diff
@@ -78,7 +80,17 @@ def email_digest(subscribed_user_ids, uid, event, **context):
 }
 
 
-def notify(uid, event, **context):
+def notify(uid, event, user, node, timestamp, **context):
+    """
+    :param uid: node's id
+    :param event: type of notification
+    :param user: user triggering notification
```

Comment:
Since `user` is an ambiguous term in this context, we might want to go further to clarify that this is the user who took the action which resulted in the notification--not the recipient of the email. Maybe we go as far as to come up with a more specific name--`agent`? Maybe not. Naming is hard.